torch/_inductor/scheduler.py

```diff
@@ -622,7 +622,7 @@ def __init__(self, scheduler: "Scheduler", snodes: List[SchedulerNode]):
         self.inverse_users = []
         self.group = max(snodes, key=lambda x: int(x.is_reduction())).group
         self.recursive_predecessors = set.union(
-            *[x.recursive_predecessors for x in snodes]
+            *[x.recursive_predecessors for x in snodes] # type: ignore[arg-type]
```

Comment:
While you are changing this, it probably should be a generator instead of a list.